azure-pipelines.yml

```diff
@@ -8,10 +8,10 @@ trigger:
 
 strategy:
   matrix:
-    php73:
```

Comment:
Since 7.3 is the lowest supported version, I'd keep it in the tests. I suggest keeping 7.4 as well since we use that in most production deployments

Reply:
Problem is that some packages do not accept <7.4... So we kind of have to.